### PR TITLE
issue: add flag to the schedule cmd to configure the `useOwnerReferencesInBackup` option  #3176

### PR DESCRIPTION
**Pull request by matheusjuvelino**
Fixes: #3176 

Signed-off-by: matheusjuvelino <matheus.juvelino@outlook.com>

### CODE REVIEW

File: pkg/cmd/cli/schedule/create.go [hunk 1]
```diff
@@ -94,6 +95,7 @@ func NewCreateOptions() *CreateOptions {
 func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	o.BackupOptions.BindFlags(flags)
 	flags.StringVar(&o.Schedule, "schedule", o.Schedule, "a cron expression specifying a recurring schedule for this backup to run")
+	flags.BoolVar(&o.UseOwnerReferencesInBackup, "use-owner-rferences-in-backup", o.UseOwnerReferencesInBackup, "specifies whether to use OwnerReferences on backups created by this Schedule")
```
Comment (zubron) [inline pkg/cmd/cli/schedule/create.go:98]: There is a typo in this flag, it should be "use-owner-r**e**ferences-in-backup".

**Review by zubron — CHANGES_REQUESTED** [review]
Thanks for getting to this so quickly! I've left one comment but other than that, this looks good!

File: pkg/cmd/cli/schedule/create.go (hunk 1 above)
Comment (matheusjuvelino) [reply pkg/cmd/cli/schedule/create.go:98]: yes, sorry for my mistake.

**Review by zubron — APPROVED** [review]
Thanks for making the changes!

**Review by nrb — APPROVED** [review]

**Pull request merged**